test/e2e/tools/eks-anywhere-test-tool/pkg/cloudwatch/cloudwatch.go

```diff
@@ -83,5 +84,17 @@ func (c Cloudwatch) getLogSegment(logGroupName string, logStreamName string, sta
 	if endTime != nil {
 		input.EndTime = endTime
 	}
-	return c.svc.GetLogEvents(input)
+	logEvents, err := c.svc.GetLogEvents(input)
+	if isInvalidParameterError(err) {
```

Comment:
I think this is should be 1 level of abstraction up (`getLogs()`). The `getLogs()` method (the consumer) should really receive the error and _decide to ignore it_. Its nuanced, I know, because there's only 1 consumer right now. To follow my rationale, consider a refactor where `getLogSegments` is called by 2 consumers where 1 wants to ignore the error but the other doesn't; as is, that wouldn't be possible.